conda/psyclone/meta.yaml

```diff
@@ -32,34 +32,12 @@ requirements:
     - python
     - six
     - sympy
+    - python-graphviz
```

Comment:
Admittedly we forgot to mention it in the release notes, but jinja2 is now also a required dependency.

Reply:
Also, can you add that fparser should be >=0.2.0?